knplabs/packagist-api/1.6/config/packages/knp_packagist_api.yaml

```diff
@@ -0,0 +1,2 @@
+services:
+    Packagist\Api\Client: null
```

Comment:
Could you elaborate on this? Dont you need it to be auto wired?

Reply:
I want to be able to autowire this serve, currently I do it like this:
https://github.com/sonata-project/dev-kit/blob/fa081ebaeaec0de4b140d87ec2b926c1d706c958/config/services.yaml#L39-L42

Reply:
You need to add: 

```yaml
services:
    Packagist\Api\Client:
        autowire: true
```

Just as you did in the `_default` section of the file you linked.

Reply:
Are you sure, when using this:
```
# config/test.yaml
services:
    Packagist\Api\Client: ~
```

it works 🤔 

![CleanShot 2020-12-03 at 15 21 54](https://user-images.githubusercontent.com/995707/101036398-4e8c5000-357b-11eb-8c96-5762a3ebac42.png)

Reply:
Oh.. I thought that class had some required constructor parameters.

Okey, then it works with the current implementation